services/movie_session.py

```diff
@@ -42,3 +43,10 @@
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
The `get` method will raise a `DoesNotExist` exception if no movie session with the given ID is found. Consider handling this exception to provide a more user-friendly error message or behavior.